docker/cloudsim_websocket/Dockerfile

```
@@ -0,0 +1,51 @@
+# Ubuntu 18.04 with nvidia-docker2 beta opengl support
+FROM ubuntu:18.04
+
+RUN export DEBIAN_FRONTEND=noninteractive \
+ && apt-get update \
+ && apt-get install -y \
+    tzdata \
+ && ln -fs /usr/share/zoneinfo/America/Los_Angeles /etc/localtime \
+ && dpkg-reconfigure --frontend noninteractive tzdata \
+ && apt-get clean
+
+RUN apt-get update -qq \
+ && apt-get install --no-install-recommends -y -qq \
+        gnupg2 \
+        lsb-release \
+        sudo \
+        wget \
+ && apt-get clean -qq
+
+# sdformat8-sdf conflicts with sdformat-sdf installed from gazebo
+# so we need to workaround this using a force overwrite
+# Do this before installing ign-gazebo
+RUN /bin/sh -c 'echo "deb [trusted=yes] http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' \
+ && /bin/sh -c 'wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -' \
+ && /bin/sh -c 'apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654'
+
+# install ign-blueprint
+RUN apt-get update \
+&&  apt-get install -y \
+    ignition-blueprint \
+ && apt-get clean
+
+# Add a user with the same user_id as the user outside the container
+# Requires a docker build argument `user_id`
+ARG user_id
+ENV USERNAME developer
+RUN useradd -U --uid ${user_id} -ms /bin/bash $USERNAME \
+ && echo "$USERNAME:$USERNAME" | chpasswd \
+ && adduser $USERNAME sudo \
+ && echo "$USERNAME ALL=NOPASSWD: ALL" >> /etc/sudoers.d/$USERNAME
+
+# Commands below run as the developer user
+USER $USERNAME
+
+# When running a container start in the developer's home folder
+WORKDIR /home/$USERNAME
+
+# Copy entry point script, and set the entrypoint
+COPY cloudsim_websocket/run_websocket.bash ./
+COPY cloudsim_websocket/cloudsim_websocket.ign ./
```

I haven't tried building this, but if the Dockerfile is in the `docker/cloudsim_websocket` directory, and assuming you're running the `docker build` command from that directory, should these paths skip the `cloudsim_websocket` directory?

```suggestion
COPY ./run_websocket.bash ./
COPY ./cloudsim_websocket.ign ./
```